### DISCUSSION TITLE
com.pieces.os default location????

### DISCUSSION THREAD

**Discussion by johnkennyuk** [pieces-app/support#369] [Q&A]
I wanted to ask, where is the default location of this dir com.pieces.os used by Pieces OS I think, I ask as I'm fairly sure it used to go to my OneDrive's KFM document's location which used to provide basic previous versioning.  But for some reason this was now stored in the local doc's location 'C:\users\username\Documents\com.pieces.os\' and not the Onedrive KFM document's location 'c:\PathToOnedrive\Documents\com.pieces.os\' also where I still find com.pieces.pfd. is this appdata path set by the Win 11 Settings - system - storage - default save location drive for documents or is it hardcoded in the app now perhaps to prevent Onedrive sync issues?  Also, would there be any difference if installed by the Suite installer that i use now?  Recently an update of Pieces OS failed meaning a reinstall was initiated from the suite installer / launcher, then I was prompted to go through the onboarding process again.  I only just realised all my old, saved content had disappeared.

On a related topic, are there any plans to add scheduling to the backup feature which I do love although it would been even better had it had scheduling and had backed up data more recently lol..

**Comment by mark-at-pieces** [comment]
Backup is in the application at the moment, as far as your old snippets would you mind scheduling a call with me https://calendar.app.google/WQAaGoEJ8kc2D3py9 and i will get you booted up
